src/Microsoft.CodeQuality.Analyzers.Exp/Core/FlowAnalysis/Analysis/NullAnalysis/NullAnalysis.cs

```diff
@@ -6,13 +6,14 @@
 namespace Microsoft.CodeAnalysis.Operations.DataFlow.NullAnalysis
 {
     using NullAnalysisData = IDictionary<AnalysisEntity, NullAbstractValue>;
-    using NullAnalysisDomain = MapAbstractDomain<AnalysisEntity, NullAbstractValue>;
+    using NullAnalysisDomain = AnalysisEntityMapAbstractDomain<NullAbstractValue>;
 
     /// <summary>
     /// Dataflow analysis to track null-ness of <see cref="AnalysisEntity"/>/<see cref="IOperation"/> instances.
     /// </summary>
     internal partial class NullAnalysis : ForwardDataFlowAnalysis<NullAnalysisData, NullBlockAnalysisResult, NullAbstractValue>
     {
+        public static readonly NullAnalysisDomain NullAnalysisDomainInstance = new NullAnalysisDomain(NullAbstractValueDomain.Default);
         private NullAnalysis(NullAnalysisDomain analysisDomain, NullDataFlowOperationVisitor operationVisitor)
```

Comment:
nit: add an empty line between

Reply:
Thanks, will make this change for next PR.